authorization_services/topics/permission-create-scope.adoc

```diff
@@ -3,7 +3,12 @@
 
 A scope-based permission defines a set of one or more scopes to protect using a set of one or more authorization policies. Unlike resource-based permissions, you can use this permission type to create permissions not only for a resource, but also for the scopes associated with it, providing more granularity when defining the permissions that govern your resources and the actions that can be performed on them.
 
+ifeval::[{project_community}==true]
+To create a new scope-based permission, select *Create scope-based permission* from the *Create permission* dropdown.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new scope-based permission, select *Scope-based* in the item list in the upper right corner of the permission listing.
+endif::[]
 
```

Comment:
"Apply Policy" on line 35 needs to be updated.   Now I see "Policy" instead.  I don't know if it is simply a name change.